src/main/java/donnafin/ui/AttributePanel.java

```diff
@@ -13,10 +10,9 @@
 /**
  * An UI component that displays information of a {@code Person}.
  */
-public class AttributePanel extends UiPart<Region> implements Attribute {
+public class AttributePanel extends UiPart<Region> {
 
     private static final String FXML = "AttributePanel.fxml";
-    private String field;
     private String value;
```

Comment:
might want to move this to line 49 so it looks neater

Reply:
Thanks! Done in 7ece3f54